examples/widgets_minimap.html

```diff
@@ -0,0 +1,129 @@
+        <!-- Import iTowns LoadingScreen and GuiTools plugins -->
+        <script src="js/GUI/LoadingScreen.js"></script>
+        <script src="js/GUI/GuiTools.js"></script>
+
+
+        <script type="text/javascript">
+
+
+
+            // ---------- CREATE A GlobeView FOR SUPPORTING DATA VISUALIZATION : ----------
+
+            // Define camera initial position
+            const placement = {
+                coord: new itowns.Coordinates('EPSG:4326', 2.351323, 48.856712),
+                range: 6000,
+                tilt: 50,
+            }
+
+            // `viewerDiv` contains iTowns' rendering area (`<canvas>`)
+            const viewerDiv = document.getElementById('viewerDiv');
+
+            // Create a GlobeView
+            const view = new itowns.GlobeView(viewerDiv, placement);
+
+            // Setup loading screen and debug menu
+            setupLoadingScreen(viewerDiv, view);
+            const debugMenu = new GuiTools('menuDiv', view);
+
+
+
+            // ---------- DISPLAY CONTEXTUAL DATA : ----------
+
+            // Add one imagery layer to the scene. This layer's properties are defined in a json file, but it could be
+            // defined as a plain js object. See `Layer` documentation for more info.
+            itowns.Fetcher.json('layers/JSONLayers/Ortho.json').then((config) => {
+                config.source = new itowns.WMTSSource(config.source);
+                view.addLayer(
+                    new itowns.ColorLayer(config.id, config),
+                ).then(debugMenu.addLayerGUI.bind(debugMenu));
+            });
+
+            // Add two elevation layers, each with a different level of detail. Here again, each layer's properties are
+            // defined in a json file.
+            function addElevationLayerFromConfig(config) {
+                config.source = new itowns.WMTSSource(config.source);
+                view.addLayer(
+                    new itowns.ElevationLayer(config.id, config),
+                ).then(debugMenu.addLayerGUI.bind(debugMenu));
+            }
+            itowns.Fetcher.json('layers/JSONLayers/WORLD_DTM.json').then(addElevationLayerFromConfig);
+            itowns.Fetcher.json('layers/JSONLayers/IGN_MNT_HIGHRES.json').then(addElevationLayerFromConfig);
+
+
+
+            // ---------- ADD MINIMAP WIDGET : ----------
+
+            // Create a ColorLayer that shall be displayed on the minimap.
+            const minimapColorLayer = new itowns.ColorLayer('minimap', {
+                source: new itowns.WMTSSource({
+                    url: 'http://wxs.ign.fr/decouverte/geoportail/wmts',
+                    name: "ORTHOIMAGERY.ORTHOPHOTOS",
+                    format: 'image/jpeg',
+                    tileMatrixSet: 'PM',
+                    crs: 'EPSG:3857',
+                    extent: new itowns.Extent(
+                        'EPSG:3857',
+                        -20026376.39, 20026376.39,
+                        -20048966.10, 20048966.10
+                    ),
```

Comment:
could use by default  world extent ?
Global extent PM is in [Extent.js](https://github.com/iTowns/itowns/blob/ec56fa9d78310afa6d581b5deff0b252266e274e/src/Core/Geographic/Extent.js#L46)